sdk/eventgrid/Azure.Messaging.EventGrid/src/EventGridPublisherClient.cs

```diff
@@ -220,19 +220,50 @@ public virtual Response SendEvent(CloudEvent cloudEvent, CancellationToken cance
         /// <param name="cloudEvents"> The set of events to be published to Event Grid. </param>
         /// <param name="cancellationToken"> An optional cancellation token instance to signal the request to cancel the operation.</param>
         public virtual async Task<Response> SendEventsAsync(IEnumerable<CloudEvent> cloudEvents, CancellationToken cancellationToken = default)
-            => await SendCloudEventsInternal(cloudEvents, true /*async*/, cancellationToken).ConfigureAwait(false);
+            => await SendCloudEventsInternal(cloudEvents, null, true /*async*/, cancellationToken).ConfigureAwait(false);
 
         /// <summary> Publishes a set of CloudEvents to an Event Grid topic. </summary>
         /// <param name="cloudEvents"> The set of events to be published to Event Grid. </param>
         /// <param name="cancellationToken"> An optional cancellation token instance to signal the request to cancel the operation.</param>
         public virtual Response SendEvents(IEnumerable<CloudEvent> cloudEvents, CancellationToken cancellationToken = default)
-            => SendCloudEventsInternal(cloudEvents, false /*async*/, cancellationToken).EnsureCompleted();
+            => SendCloudEventsInternal(cloudEvents, null, false /*async*/, cancellationToken).EnsureCompleted();
```

Comment:
Out of curiosity, why use a comment rather than the named variable syntax?

Reply:
No particular reason - I just copied it from an existing method.